src/Models/DemoRepository.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace Tricentis.RestApiTemplate.Models;
+
+public class DemoRepository : IDemoRepository
+{
+
+    private readonly DemoContext _demoContext;
+    private DbSet<DemoItem> _items;
+
+    public DemoRepository(DemoContext dataContext)
+    {
+        _demoContext = dataContext;
```

Comment:
Since `dataContext.Items` is what you always access, you may wanna do a null check here and throw exception if null.
```suggestion
        _demoContext = dataContext ?? throw new ArgumentNullException(nameof(dataContext));
```